contract/kernel/kernel.go

```diff
@@ -200,6 +200,22 @@ func (k *Kernel) CreateBlockChain(name string, data []byte) error {
 		os.RemoveAll(fullpath)
 		return err
 	}
+	tx, err := utxo.GenerateRootTx(data)
+	if err != nil {
+		k.log.Warn("GenerateRootTx error", "fullpath", fullpath)
+		os.RemoveAll(fullpath)
+		return err
+	}
+	txlist := []*pb.Transaction{tx}
+	k.log.Trace("Start to ConfirmBlock")
+	b, err := ledger.FormatRootBlock(txlist)
+	if err != nil {
+		k.log.Warn("format block error")
```

Comment:
It's better to print the error message in Warning log since this err info is not returned.